ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/radiobutton.html

```diff
@@ -0,0 +1,53 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2022 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<sly data-sly-use.renderer="${'radiobutton.js'}"
+     data-sly-use.clientlib="${'/libs/granite/sightly/templates/clientlib.html'}"
+     data-sly-use.label="${renderer.labelPath}"
+     data-sly-use.shortdescription="${renderer.shortDescriptionPath}"
+     data-sly-use.longdescription="${renderer.longDescriptionPath}"
+     data-sly-use.errormessage="${renderer.errorMessagePath}"
+     data-sly-use.questionmark="${renderer.questionMarkPath}" />
+<div data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"
+     data-sly-use.radiobutton="com.adobe.cq.forms.core.components.models.form.RadioButton"
+     data-sly-use.component="com.adobe.cq.wcm.core.components.models.Component"
+     class="cmp-adaptiveform-radiobutton"
+     data-cmp-is="adaptiveFormRadioButton"
+     id="${component.id}"
+     data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}">
+    <div data-sly-call="${label.label @componentId=component.id, labelValue=radioButton.label.value, labelVisible=radioButton.label.visible, bemBlock='cmp-adaptiveform-radiobutton'}" data-sly-unwrap></div>
+    <div data-sly-call="${questionMark.questionMark @componentId=component.id, longDescription=radioButton.description, bemBlock='cmp-adaptiveform-radiobutton'}" data-sly-unwrap></div>
+
+    <div data-sly-list="${radioButton.enums}" data-sly-unwrap>
+        <div class="cmp-adaptiveform-radiobutton__option ${radioButton.orientation}">
+            <label class="cmp-adaptiveform-radiobutton__option__label" aria-label="${radioButton.enumNames[itemList.index]}">
+                <input type="radio"
+                       name="${radioButton.name}"
+                       class="cmp-adaptiveform-radiobutton__option__widget"
+                       id="${component.id}${'_'}${itemList.index}__widget"
+                       value="${item.toString}"
+                       aria-describedby="${radioButton.labelForId}_desc"
+                       checked="${radioButton.enums[itemList.index] == radioButton.default[0]}"
+                       aria-required="${radioButton.required}">
+                <span>${radioButton.enumNames[itemList.index]}</span>
```

Comment:
On click of the radio button caption, the widget should work. Can you check that?